docs/First-Installation.md

```diff
@@ -113,49 +115,30 @@ For more information, see [Hotkeys](RetroArch-Configuration#hotkeys)
-
 ## Transferring ROMs
 
 You will not see any game systems (NES, n64, Playstation, etc) on the main menu until you add ROMs! Visit the [Transferring ROMs](Transferring-Roms) page to learn how to transfer ROMs to RetroPie.
 
-## AUDIO
+## Audio
 
-In general RetroPie audio will work out of the box without any tweaking, but if you have audio issues you should follow the instructions on the [Sound Issues Page](Sound-Issues) to fix them. You will most likely need to visit the [Sound Issues Page](Sound-Issues) if you are using a USB Audio device, or if you are using an aftermarket RPi HAT add-on audio device (such as a Justboom sound card).
+In general RetroPie audio will work out-of-the-box without any tweaking if using HDMI, but if you have audio issues you should follow the instructions on the [Sound Issues Page](Sound-Issues) to fix them. You will most likely need to visit this page if you are using a USB audio device, or if you are using an aftermarket RPi HAT add-on audio device (such as a Justboom sound card).
 
 ## PLAY!
 
-After you've added your roms you need to restart EmulationStation in order for them to show up. You can restart EmulationStation from the start menu, or by rebooting your pi with `sudo reboot`. 
+After you've transferred your ROMs, you need to restart EmulationStation in order for them to show up. You can restart EmulationStation by pressing Start > Quit > Restart EmulationStation, or with SSH access by rebooting your pi with `sudo reboot`. Once rebooted, you should see the game systems appear on the main menu alongside the original "RetroPie" menu.
+
+## Additional Setup Options
+
+* [Configure Wifi](Wifi) to allow for easy [Updating](Updating-RetroPie), downloading themes, and SSH
+* [Enable SSH](SSH) to connect to your Raspberry Pi remotely and make it easy for [Transferring ROMs](Transferring-Roms)
+* Configure more controllers. This can be done after plugging in the new controller and pressing *Start* on your controller and selecting *Configure Input*
+* Install additional emulators/cores. From the RetroPie main menu, select *RetroPie Setup > Manage Packages > Manage Optional Packages*. Stable additional packages are available here with more unstable packages listed in the *Manage experimental packages* section. The packages are ordered first by type (emulators / Libretro cores / ports), then alphabetically. By selecting a package you can choose to install it or remove it. Some packages also have additional configurations.
```

Comment:
Again I think this info is best on the appropriate page: https://retropie.org.uk/docs/Updating-RetroPie/#updatinginstalling-individual-packages - I think a link is enough